js-html-games/bogscript.js

```diff
@@ -1,7 +1,378 @@
-console.log("hello");
-// fetch('dictionary.txt')
-//   .then(response => response.text())
-//   .then(data => {
-//   	// Do something with your data
-//   	console.log(data);
-//   });
\ No newline at end of file
+const grid = document.getElementById("boggle");
+const word = document.getElementById("word");
+const wordForm = document.getElementById("wordForm");
+const error = document.getElementById("errorMessage");
+const restart = document.getElementById("restart");
+const numR = 6;
+const numC = 4;
+const maxGuess = numR;
+let currentRow = 0;
+const keyboardLetters = "QWERTYUIOPASDFGHJKLZXCVBNM";
+const alphabet = (keyboardLetters.split("").sort().join(""));
+let typedInput = [];
+let letterArray = [];
+let url = "https://raw.githubusercontent.com/dwyl/english-words/master/words_alpha.txt";
+
+function createGrid(rows, cols) { // generates grid with random letters
+    for (let i = 0; i < rows; i++) {
+        const row1 = document.createElement('div');
+        row1.setAttribute('class', "line");
+        grid.appendChild(row1);
+        function createCols() {
+            for (let j = 0; j < cols; j++) {
+                const col1 = document.createElement('div');
+                const letter = document.createElement('p');
+                col1.setAttribute('class', "letter");
+                // letter.innerHTML = randLetter();
+                col1.appendChild(letter);
+                row1.appendChild(col1);
+            }
+        }
+        createCols();
+        // const allDivs = document.querySelectorAll(".letter");
+        // for (let i = 0; i < allDivs.length; i++) {
+        //     allDivs[i].innerText = i + 1;
+        // }
+    }
+}
+
+function createKeyboard(rightWord, wordList) {
+    const enter = document.createElement('button');
+    enter.setAttribute('class', "enter");
+    enter.innerText = "Enter";
+    enter.addEventListener('click', function () {
+        matchWord(rightWord, typedInput.join(""), wordList)
+        typedInput = [];
+        console.log(typedInput.join(""));
+    })
+    const back = document.createElement('button');
+    back.setAttribute('class', "back");
+    back.innerText = "Backspace";
+    back.addEventListener('click', function () {
+        erase();
+        typedInput.pop();
+    })
+    for (let i = 0; i < keyboardLetters.length; i++) {
+        const alphaLetter = document.createElement('button');
+        alphaLetter.setAttribute('class', "alphaLetter");
+        alphaLetter.innerText = keyboardLetters[i];
+        alphaLetter.addEventListener('click', function () {
+            typedInput.push(keyboardLetters[i]);
+            type();
+            console.log(typedInput);
+            console.log(keyboardLetters[i]);
```

Comment:
would be a good habit to keep: removing console.log's before merging / pushing code. i know that you might still be developing/working on this proj tho